project-fortis-interfaces/src/components/Admin/Admin.js

```diff
@@ -129,7 +134,10 @@ class Admin extends React.Component {
                       </div>
                     </TabPanel>
                     <TabPanel>
-                      <h2>Blacklisted Terms</h2>
+                      <h2>Blacklisted Terms <FontIcon className="fa fa-question" data-tip data-for='blacklist' data-place='right' color={blue500} hoverColor={blue700}/></h2>
+                      <ReactTooltip id='blacklist'>
+                        <span>Provide a comma delimited list of blacklist terms, i.e. hunger, flood, hurricane</span>
```

Comment:
Add to tooltip something like "If an event matches _all_ these terms, it will be discarded."

Reply:
ok!